phy/restart_wt.F

```diff
@@ -330,10 +330,14 @@ subroutine restart_wt
         call wrtrst('ficem_da',trim(c5p)//' k2 time',ficem_da,ip)
         call wrtrst('abswnd_da',trim(c5p)//' k2 time',abswnd_da,ip)
         call wrtrst('atmco2_da',trim(c5p)//' k2 time',atmco2_da,ip)
+        call wrtrst('atmbrf_da',trim(c5p)//' k2 time',atmbrf_da,ip)  ! not read in restart_rd, necesarry?
```

Comment:
@matsbn @tjiputra 

Is it necessary that atmbrf_da is written to restart? If it is, then it needs to be also _read_ from the file (which is not the case now). For a fully prognostic variable it is probably necessary? Also, if it is necessary, then it would be also necessary for DMS?